### PR TITLE
- Update json_stream initialization to improve compatibility

### PR DESCRIPTION
**Pull request by theblindprogramm3r**

### CODE REVIEW

File: src/loader/manifest_file.cpp [hunk 1]
```diff
@@ -721,7 +721,7 @@ ApiLayerManifestFile::~ApiLayerManifestFile() = default;
 
 void ApiLayerManifestFile::CreateIfValid(ManifestFileType type, const std::string &filename,
                                          std::vector<std::unique_ptr<ApiLayerManifestFile>> &manifest_files) {
-    std::ifstream json_stream = std::ifstream(filename, std::ifstream::in);
+    std::ifstream json_stream(filename, std::ifstream::in);
```
Comment (rpavlik) [inline src/loader/manifest_file.cpp:724]: Is the `std::ifstream::in` still required given that it's using a `std::ifstream`?

**Review by rpavlik — CHANGES_REQUESTED** [review]
One minor question, plus we need the CLA

File: src/loader/manifest_file.cpp (hunk 1 above)
Comment (theblindprogramm3r) [reply src/loader/manifest_file.cpp:724]: Its not required, but by convention, it is recommended. Ex. if the default ever changes (not likely) from ::in to something else.

Comment (theblindprogramm3r) [reply src/loader/manifest_file.cpp:724]: Ex. the default signature for ifstream:
explicit ifstream (const char* filename, ios_base::openmode mode = ios_base::in);

Comment (rpavlik) [reply src/loader/manifest_file.cpp:724]: Interesting. I don't think I had used it with ifstream/ofstream in literally years. OK.

**Review by rpavlik — APPROVED** [review]

**Comment by rpavlik** [comment]
Hmm, I clicked the recheck but it still says pending . Can you click the cla recheck link?

**Comment by theblindprogramm3r** [comment]
> Hmm, I clicked the recheck but it still says pending . Can you click the cla recheck link?

I've gone back and done it, when I click on the link the top of the page says "You have signed the ...." and no longer offers a button to sign.

**Pull request merged**